LFGBulletinBoard/dungeons/classic.lua

```diff
@@ -150,7 +150,7 @@ local infoOverrides = {
         size = 20,
     },
     -- Demon Fall Canyon
-    DFC = isSoD and {
+    DFC = {
```

Comment:
This ones a minor nit.

Because `DFC` is already conditionally added to the `LFGDungeonID`'s table, then if `isSoD==false` this check is never hit. 

The entries above this on, for Gnomer/BFD/ST, use the `isSoD` conditional to modify the original data provided for those dungeons. They are considered dungeons not raids and have a different level ranges in regular Classic Era, so the `isSoD` check is used to modify that data on SoD clients only.

Reply:
Might be worth keeping as an extra safety check